backend/driver.pm

```diff
@@ -28,14 +28,9 @@ use IO::Select;
 use POSIX '_exit';
 require IPC::System::Simple;
 use autodie ':all';
+use bmwqemu qw(fctwarn logdie fctdbg);
 use myjsonrpc;
 
-# TODO: move the whole printing out of bmwqemu
```

Comment:
Yep, moving the logging to something like `OpenQA::Log` instead

Leaving this for future reference

```perl
package OpenQA::Log
use Log::Log4perl

package bmwqemu
use base OpenQA::Log
```